src/Microsoft.Data.SqlClient/netcore/src/Microsoft/Data/SqlClientX/IO/TdsStream.cs

```diff
@@ -50,9 +52,13 @@ internal class TdsStream : Stream, ITdsWriteStream
             set => _writeStream.PacketHeaderType = value; 
         }
 
-        public TdsStream(Stream underLyingStream, TdsWriteStream writeStream) : base()
+        /// <inheritdoc />
+        public int Spid => _readStream.Spid;
+
+        public TdsStream(TdsWriteStream writeStream, TdsReadStream readStream) : base()
```

Comment:
Comment :)

Reply:
Cmonnnnnn ... Its a constructor 😄 
I will add comments

Reply:
I have PTSD from DSCT's overly strict rules about comments. Fortunately I could just click "Add standard comment" and it would be happy. And the standard comment was something like "Constructs and initialized a new instance of <see cref="TdsStream" />". I won't complain too hard about not including such comments, and I definitely will push for turning off the "Comments must end with `.`" rule, if we ever adopt codeanalysis rules.

Reply:
BTW @benrr101  my comment about "Its a constructor" was not a serious one. 

I have a strong belief that code is read a lot more than it is written. So it is always good to over-comment than not. So keep the feedback coming. There is a reason I address them as well 😄 

I think we need to think of a solution to enforce the code style/comment etc during the build of the code so that the code author can be asked to improve the code with builds. This way the reviewer doesn't have a cognitive overload of finding problems with the style of code. And this needs to be done in a way that only new code is flagged, and the old code stays.

Honestly, I wished that each field of TdsParserStateObject was documented, or documented better. Otherwise we keep reverse engineering the intentions.